src/views/RequiredSignoffs/ViewSignoff/index.jsx

```diff
@@ -112,9 +119,19 @@ function ViewSignoff({ isNewSignoff, ...props }) {
   };
 
   // TODO: Add save logic
-  const handleSignoffSave = () => {};
+  const handleSignoffSave = async () => {
+    // Call the save endpoint with the appropriate parameters
```

Comment:
So we'll probably want another function in https://github.com/mozilla-frontend-infra/balrog-ui/blob/master/src/services/requiredSignoffs.js to do the actual work of saving?

Reply:
Yeah, that's correct.